api/test/unit/utilities/generate-single-use-code.spec.ts

```diff
@@ -1,22 +1,7 @@
-import { Test, TestingModule } from '@nestjs/testing';
-import { INestApplication } from '@nestjs/common';
 import { generateSingleUseCode } from '../../../src/utilities/generate-single-use-code';
-import { AppModule } from '../../../src/modules/app.module';
 
 describe('generateSingleUseCode', () => {
-  let app: INestApplication;
-
-  beforeAll(async () => {
-    const moduleFixture: TestingModule = await Test.createTestingModule({
-      imports: [AppModule],
-    }).compile();
-
-    app = moduleFixture.createNestApplication();
-    await app.init();
-  });
   it('should generate mfa code of the specified length', () => {
-    expect(generateSingleUseCode().length).toEqual(
-      Number(process.env.MFA_CODE_LENGTH),
-    );
+    expect(generateSingleUseCode(5).length).toEqual(5);
```

Comment:
Should we still be checking for the env var?

Reply:
I ended up running into some issues with this test (mainly flakiness, sometimes passing, sometimes failing)

so what I ended up doing was changing the `generateSingleUseCode()` to take in a numeric argument. In the code we now send `Number(process.env.MFA_CODE_LENGTH)` wherever we call `generateSingleUseCode` so no need to check this function. Just make sure the length matches what the incoming arg says the length should be